pkg/controller/controller.go

```diff
@@ -334,6 +343,36 @@ func ttls() []string {
 	return []string{"15m", "30m", "1h", "2h", "4h"}
 }
 
-func categories() []string {
-	return []string{"explanation"}
+func (c *Controller) categories() []string {
+	validators := c.p.Validators()
+	categories := make([]string, 0, len(validators))
+	for v := range validators {
+		categories = append(categories, v)
+	}
```

Comment:
https://pkg.go.dev/golang.org/x/exp/maps#Keys

Reply:
Actually I'd suggest creating the map inline.

```go
for v := range validators {
  categories = append(categories, Pair{Key: v, Text: strings.ToUpper(v)}
}
```

Reply:
I have been playing with different functions under `strings` package, but don't seem a way to make things camel case.  Note that @qiaoweiz is working on passing back human-friendly category and hint from validator plugin, as a result I simply add a getText function here to stay compatible with existing code.

Reply:
Will use ToTitle in html for now after merging in #303